doc/quick_start_guide.rst

```diff
@@ -126,18 +126,37 @@ Building ``dpnp`` for these targets requires that these CodePlay plugins be inst
 installation layout of compatible version. The following plugins from CodePlay are supported:
 
     - `oneAPI for NVIDIA(R) GPUs <codeplay_nv_plugin_>`_
+    - `oneAPI for AMD GPUs <codeplay_amd_plugin_>`_
 
 .. _codeplay_nv_plugin: https://developer.codeplay.com/products/oneapi/nvidia/
+.. _codeplay_amd_plugin: https://developer.codeplay.com/products/oneapi/amd/
 
 Building ``dpnp`` also requires `building Data Parallel Control Library for custom SYCL targets.
 <https://intelpython.github.io/dpctl/latest/beginners_guides/installation.html#building-for-custom-sycl-targets>`_
 
-Build ``dpnp`` as follows:
+``dpnp`` can be built for CUDA devices as follows:
 
 .. code-block:: bash
 
     python scripts/build_locally.py --target=cuda
 
+And for AMD devices:
+
+.. code-block:: bash
+
+    python scripts/build_locally.py --target-hip=gfx90a
```

Comment:
In general it might be unclear what `gfx90a` means here. It'd be great to clarify.